reviews/utils/review_utils.py

```diff
@@ -304,6 +304,21 @@ def notify_secretary(decision):
     msg_plain = render_to_string('mail/decision_notify.txt', params)
     send_mail(subject, msg_plain, settings.EMAIL_FROM, [secretary.email])
 
+def notify_secretary_all_decisions(review):
+    """
+    Notifies a secretary all Decisions have been made for a certain review
+    """
+    secretary = get_secretary()
+    subject = "FETC-GW {}: alle beoordelingen toegevoegd".format(
+        review.proposal.committee_prefixed_refnum(),
+    )
+    params = {
+        "secretary": secretary.get_full_name(),
+        "proposal": review.proposal,
+        "decisions": review.decision_set.all()
+    }
+    msg_plain = render_to_string("mail/all_decisions_notify.txt", params)
```

Comment:
Any translations existing in this input, e.g. `decision.get_go_display`, will render to the currently selected language.

If the user who triggered the `update_go()` happened to be set to English, all translations in here will also turn into English, which is a bit weird. 

See `proposal_utils.notify_local_staff()` for an example of how we manually activate a language for email sending, and then reset it afterwards.